servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -812,31 +781,7 @@
             "description": "The report of the node health",
             "required": true,
             "schema": {
-              "type": "object",
-              "description": "The report of the node health",
-              "properties": {
-                "SourceId": {
-                  "type": "string"
-                },
-                "Property": {
-                  "type": "string"
-                },
-                "HealthState": {
-                  "$ref": "#/definitions/HealthState"
-                },
-                "Description": {
-                  "type": "string"
-                },
-                "TimeToLiveInMilliSeconds": {
-                  "type": "string"
-                },
-                "SequenceNumber": {
-                  "type": "string"
-                },
-                "RemoveWhenExpired": {
-                  "type": "boolean"
-                }
-              }
+              "$ref": "#/definitions/NodeHealthReport"
```

Comment:
Edit: DeployedServiceHealthReport, ApplicationHealthReport, ServiceHealthReport, PartitionHealthReport, ReplicaHealthReport, ClusterHealthReport, NodeHealthReport - all have the same schema/structure (same model properties). What is the benefit by creating  7 models when this could be achieved by creating 1 model named "HealthReport" that is referenced in 7 places?